### PR TITLE
[4.0] Add open new window tooltip

### PR DESCRIPTION
**Pull request by Quy**
Continuation of #27993 .

### Summary of Changes
Add "Open new window" tooltip.

### Testing Instructions
Go to System > Administrator Template Styles
Click Options button
Enable `Preview Module Positions`
Click Save & Close button
Hover mouse over icon under Preview column.
See tooltip

### Actual result BEFORE applying this Pull Request
No tooltip

### CODE REVIEW

File: administrator/components/com_templates/tmpl/styles/default.php [hunk 1]
```diff
@@ -85,6 +85,7 @@
 										<?php $client = (int) $item->client_id === 1 ? 'administrator' : 'site'; ?>
 										<!-- external link icon generated by css -->
 										<a href="<?php echo Route::link($client, 'index.php?tp=1&templateStyle=' . (int) $item->id); ?>" target="_blank" class="jgrid">
```
Comment (richard67) [inline administrator/components/com_templates/tmpl/styles/default.php]:
```suggestion
										<a href="<?php echo Route::link($client, 'index.php?tp=1&templateStyle=' . (int) $item->id); ?>" target="_blank" class="jgrid"
```

Comment (richard67) [reply administrator/components/com_templates/tmpl/styles/default.php]: j4-pr29971

**Review by richard67 — CHANGES_REQUESTED** [review]
Fix HTML syntax

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 98620b67eb5724b13894240d57abbd0180710691<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29971">issues.joomla.org/tracker/joomla-cms/29971</a>.</sub>

**Comment by adj9** [comment]
I have tested this item :white_check_mark: successfully on 98620b67eb5724b13894240d57abbd0180710691<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29971">issues.joomla.org/tracker/joomla-cms/29971</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29971">issues.joomla.org/tracker/joomla-cms/29971</a>.</sub>

**Pull request merged**